xml/_types_test.ts

```diff
@@ -1 +1,18 @@
 import "./_types.ts"
+import { stringify } from "./mod.ts"
+
+// Verify that the generic type is correctly inferred.
+const _example = stringify({
+  "@version": "1.1",
+  "@encoding": "UTF-8",
+  "@standalone": "yes",
+  "#doctype": {
+    "~name": "html",
+    "~children": [],
+    "#text": "",
+  },
+  "#instructions": {},
+  "~name": "html",
+  "~children": [],
+  "#text": "",
+})
```

Comment:
I think the CI is failing because it's missing an assertion for the root node (not related to your actual pr).
Currently it only checks if there are multiple root node:
https://github.com/lowlighter/libs/blob/52d3ad2fd621ffa6fcc706d42ce43e7d8b5c3de7/xml/stringify.ts#L88-L93
But not if there is at least one. Since your example doesn't contain any "root node" it's undefined and that's what is passed to `xml_node()`. Adding `root: {}` or `root: null` (or any non-special property) should fix it

I think since the test is mostly to test the generic type, it's better to remove all optional properties (like `@standalone`, `#doctype`, etc.)

Reply:
Yeah, I fixed it locally but didn't push b/c I was investigating the Lume code.